semcore/switch/CHANGELOG.md

```diff
@@ -2,6 +2,18 @@
 
 CHANGELOG.md standards are inspired by [keepachangelog.com](https://keepachangelog.com/en/1.0.0/).
 
+## [5.28.0] - 2024-06-11
+
+### Changed
+
+- set as deprecated `hidden` property on Switch.Addon.
```

Comment:
Maybe we should better explain why we did it here?

Reply:
fixed